### PR TITLE
Vulnerability a8

### PR DESCRIPTION
**Pull request by lousander**
To start the app you should set the following environment variables:
A8_ADMIN_USER
A8_ADMIN_PASS

### CODE REVIEW

**Comment by lousander** [comment]
I just want to share that one of my first searches on google while trying to mitigate this vulnerability was "pickle cookies", and I ended up with a lot of recipes... apparently [Pickle Cookies](https://www.google.com/search?q=pickle+cookies&oq=pickle+cookies) are actually a thing! :scream:

File: owasp-top10-2017-apps/a8/amarelo-designs/deployments/docker-compose.yml [hunk 1]
```diff
@@ -13,6 +13,10 @@ services:
     networks: 
       - a8_net
     restart: always
+    environment:
+      - A8_ADMIN_USER=$A8_ADMIN_USER
+      - A8_ADMIN_PASS=$A8_ADMIN_PASS
+      - A8_SECRET="bsyE4Cav7yJc3sJCD4Lm"
```
Comment (Krlier) [inline owasp-top10-2017-apps/a8/amarelo-designs/deployments/docker-compose.yml]: 👀

Comment (rafaveira3) [reply owasp-top10-2017-apps/a8/amarelo-designs/deployments/docker-compose.yml]: 🧐

**Comment by lousander** [comment]
Cool @Krlier, thanks for the tip! :)

**Pull request closed without merging**